src/common/components/floating-faq/index.scss

```diff
@@ -0,0 +1,299 @@
+.help-btn {
+  position: fixed;
+  z-index: 900;
+  right: 2rem;
+  bottom: 2rem;
+  background: $dark-sky-blue;
+  color: $white;
+  border-radius: 20px 2px 20px 20px !important;
+  padding: 10px 20px 10px 10px !important;
```

Comment:
Please use `@include button-variant` mixin for button generation. It will help you to generate hover/active states

Reply:
@MTSaeed Check what @dkildar commented above ^

Reply:
@feruzm Updated.